x/staking/types/keys.go

```diff
@@ -53,6 +56,14 @@ var (
 
 	HistoricalInfoKey   = []byte{0x50} // prefix for the historical info
 	ValidatorUpdatesKey = []byte{0x61} // prefix for the end block validator updates key
+
+	TokenizeShareRecordPrefix          = []byte{0x71} // key for tokenizeshare record prefix
```

Comment:
This key conflicts with another key that exists in the future Cosmos SDK releases:
https://github.com/cosmos/cosmos-sdk/blob/dd585bac075596cc075643e36366e4ed2381c6c9/x/staking/types/keys.go#L61

My suggestion is similar to error codes, just move these into own subspace - starting from `0x80`:

```go
TokenizeShareRecordPrefix          = []byte{0x81} // key for tokenizeshare record prefix
TokenizeShareRecordIDByOwnerPrefix = []byte{0x82} // key for tokenizeshare record id by owner prefix
TokenizeShareRecordIDByDenomPrefix = []byte{0x83} // key for tokenizeshare record id by denom prefix
LastTokenizeShareRecordIDKey       = []byte{0x84} // key for last tokenize share record id
TotalLiquidStakedTokensKey         = []byte{0x85} // key for total liquid staked tokens
TokenizeSharesLockPrefix           = []byte{0x86} // key for locking tokenize shares
TokenizeSharesUnlockQueuePrefix    = []byte{0x87} // key for the queue that unlocks tokenize shares
```

Reply:
updated to your suggestions [here](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/7b4eac53bae3069f0db6de4fbf0df64354fb9711)